export.py

```diff
@@ -250,13 +301,12 @@ def downloadCourseFiles(course, course_view):
         for file in files:
             file_folder=course.get_folder(file.folder_id)
             
-            folder_dl_dir=os.path.join(dl_dir,makeValidFolderPath(file_folder.full_name))
+            folder_dl_dir=os.path.join(dl_dir, makeValidFilename(file_folder.full_name))
```

Comment:
Reason for this change?

Reply:
Could we remove or merge the two validation functions?

Reply:
Oops I was testing out whether makeValidFolderPath was needed and I must've left that changed. I think in that case it might make sense to keep it makeValidFolderPath.

Reply:
Sounds good.